python/nano/src/bigdl/nano/deps/neural_compressor/inc_api_2.py

```diff
@@ -144,8 +191,7 @@ def new_collate_fn(batch):
         accuracy_criterion = AccuracyCriterion(
             higher_is_better=accuracy_criterion.get('higher_is_better', True),
             criterion=criterion,
-            # we calculate `tolerable_loss` by `1 - target_accuracy`
-            tolerable_loss=1.0 - accuracy_criterion[criterion]
+            tolerable_loss=float(accuracy_criterion[criterion])
```

Comment:
shall we rename `accuracy_criterion` to `loss_criterion` ?

Reply:
I think we can still use `accuracy_criterion` as we have detailed comment `:param accuracy_criterion:  Tolerable accuracy drop, defaults to None meaning no accuracy control.`